hangman.rb

```diff
@@ -0,0 +1,149 @@
+
+class Hangman
+	def initialize
+
+		# Library of words
+		@words = ["orca", "cheese", "cake", "pumpkin", "sun", "apple", "cloud", "paint", "bird", "travel", "lollipop", "ada"]
+
+		@answer_word = choose_word # chooses answer (array of characters)
+		@answer_copy = @answer_word.dup # creates arrage of correct length for @word_display
+		@word_display = @answer_copy.fill("_")
+		@letters_guessed = []
+
+		@default_guesses = 7
+		@wrong_count = 0
+
+		start
+	end
+
+	def start
+		@keep_playing = true
+		puts "Let's play HANGMAN!"
+
+		update_visuals
+		play
+	end
+
+	def play
+		# Checks for win or lose
+		while @keep_playing == true
+			if @wrong_count < 7 && win != true
+				puts "Guess one letter and then hit enter. Type \"quit\" if you don't want to play."
+				print "YOUR GUESS: "
+				@current_guess = get_guess # gets guess from user
```

Comment:
Based on the way the guess is being returned from `get_guess` and passed into `check_guess` you should probably just use a local variable here rather than an instance variable